src/test/java/org/elasticsearch/action/count/CountRequestBuilderTests.java

```diff
@@ -78,7 +78,7 @@ public void testStringQueryToString() {
         CountRequestBuilder countRequestBuilder = client.prepareCount();
         String query = "{ \"match_all\" : {} }";
         countRequestBuilder.setQuery(new BytesArray(query));
-        assertThat(countRequestBuilder.toString(), containsString("\"query\":{ \"match_all\" : {} }"));
+        assertThat(countRequestBuilder.toString(), containsString("\"query\" : {\n    \"match_all\" : { }\n  }"));
```

Comment:
So that is because it didn't use pretty print before but it does now? I think we should keep the test but fix it. toString using prettyPrint sounds good, cause then it is readable. Maybe use the system line separator in the test?